watertap/examples/flowsheets/case_studies/anaerobic_digestor/Translator_BSM2.py

```diff
@@ -0,0 +1,131 @@
+###############################################################################
```

Comment:
I'm uncertain about the need for this file. It seems like the test has what is needed, so maybe we don't need this example. Any reasoning I am missing?

Reply:
I was showing the implementation of the translator block for users, I will not include it in this PR and add it when we do the PR on the other translator to show users